tests/system/providers/google/cloud/bigtable/example_bigtable.py

```diff
@@ -63,24 +63,35 @@
 from airflow.utils.trigger_rule import TriggerRule
 from tests.system.providers.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID").lower().replace('_', '-')
+
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
-DAG_ID = "bigtable"
+DAG_ID = f"bigtable"
```

Comment:
I think we don't need f-string here
```suggestion
DAG_ID = "bigtable"
```